google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginUsersPanel.java

```diff
@@ -186,11 +187,9 @@ public void mouseDragged(MouseEvent event) {
     boolean noUsersAvailable =
         (listModel.getSize() == 1) && (listModel.get(0) instanceof NoUsersListItem);
     JButton addAccountButton = new JButton(noUsersAvailable ? SIGN_IN : ADD_ACCOUNT);
-    addAccountButton.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(ActionEvent event) {
-        Services.getLoginService().logIn();
-      }
+    addAccountButton.addActionListener(event -> {
+      SwingUtilities.getWindowAncestor(this).dispose();
```

Comment:
Hmm, well this is unrelated anyway so I can file a separate bug and remove this from this PR if you'd like.